test/fixtures/flags-tasks-with-nowrap.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var gulp = require('gulp');
+
+gulp.task('task1', function() {});
+gulp.task('task1').description = 'Description for gulp.task("task1")';
+
+gulp.task('task2', function() {});
+if (!gulp.task('task2').unwrap) {
+  var fn2 = function() {};
+  gulp.task('task2').unwrap = function() {
+    return fn2;
+  };
+}
+gulp.task('task2').unwrap()
+  .description = 'Description for gulp.task("task2").unwrap()';
+
+gulp.task('task3', function() {});
```

Comment:
It might be my mistake or I couldn't remember the reaseon. I'll remove one line of them and one more line about 'no-desc'.